downstream/assemblies/platform/assembly-install-aap-operator.adoc

```diff
@@ -11,6 +11,10 @@ ifdef::context[:parent-context: {context}]
 
 .Prerequisites
 * You have installed the {PlatformName} catalog in Operator Hub.
+* You have created a Storage Class if one does not exist. See link:https://docs.openshift.com/container-platform/4.10/storage/dynamic-provisioning.html[Dyamic Provisioning] for details.
```

Comment:
Do we need to specify which StorageClass object definition to use?

Reply:
@ariordan-redhat All good questions. According to Christian, he's not sure what the defaults SC should be and said we should point the users to these sources in the short term and then "Long term, it would be good for us to take the time to figure out a recommended path and document steps."